utils/kernel-parser.c

```diff
@@ -4,6 +4,7 @@
 #include <string.h>
 #include <sys/mman.h>
 #include <sys/stat.h>
+#include <unistd.h>
```

Comment:
Fix warning: implicit declaration of function 'close' #1859

Reply:
Nice. You can also add it as `Fixed: #1553, #1859` in the commit message.

Reply:
As @namhyung mentioned at https://github.com/namhyung/uftrace/issues/1859#issuecomment-1857471556, we better make this change in a separate PR. Please drop the change here.